src/crimson/net/SocketConnection.h

```diff
@@ -40,6 +40,13 @@ class SocketConnection : public Connection {
   Dispatcher& dispatcher;
   seastar::gate pending_dispatch;
 
+  /// my_socket_port can be different from my_addr.get_port() if is
+  /// connector.
```

Comment:
* fixed the comment style in this PR.

* For peers, `is_server_side()` will always return `False`, so we cannot use a single `client_port` in this scenario: https://github.com/ceph/ceph/blob/master/src/msg/Policy.h#L62-L67

  There is no client/server for messengers as peers. But in the context of established connection/socket, there's always be a side which initiates the connection, who I call it the connector side, and another called the acceptor side. I thought it might be better to distinguish this from the messenger client/server concept.

Reply:
i see. for OSD-OSD connections, both sides are "peers" in the sense that they are equal parties. what i don't like is that `SocketConnection::print()` prints both ports even if one of them are always `0`. how about having a scoped enum member variable noting if the connection is established with "connect" or "accept" ? we can set it in `start_connect()` and `start_accept()`.

Reply:
done.